pillar/heroku/ocw-studio.sls

```diff
@@ -70,6 +74,7 @@ heroku:
   api_key: __vault__::secret-operations/global/heroku/api_key>data>value
   config_vars:
     ALLOWED_HOSTS: '["*"]'
+    API_BEARER_TOKEN: __vault__::secret-open-courseware/ocw-studio/{{ environment }}/api-bearer-token>data>value
```

Comment:
Made a guess here, it will have to be created in vault

Reply:
What is this used for?

Reply:
Authenticating against a new API webhook endpoint (`Authentication: Bearer <API_BEARER_TOKEN>`) to make sure it's coming from a legitimate source.  The pipeline will be sending a request to this endpoint on either success or failure to notify site admins.

Reply:
I'm assuming you mean an endpoint in the OCW site? Ideally we won't write that token into the pipeline definition itself and instead use the var syntax to fetch it from Vault instead.

Reply:
I can make it work that way, but it needs to be available to OCW Studio too so that it can [make sure they match](https://github.com/mitodl/ocw-studio/pull/381/files#diff-16563ed2f0c515e79c29ce5e9eabd02fe39202da9bee751da6bdb640d1ab600bR330).

Reply:
Let me know which vault var name it will be and I will update the pipeline definition in the OCW Studio PR to use it.

Reply:
I have not added the token value. What format is the app expecting for that? e.g. length requirements/constraints, UUID vs random string, etc.

As to the draft and live URL those aren't set currently but eventually that will be ocw.mit.edu for live. I'd defer to @pdpinch for what we want to use currently.

Reply:
It will be a bearer token sent in the header so it can be any hard-to-guess string of any reasonable length.

Reply:
A uuid would be fine but not strictly necessary